### DISCUSSION TITLE
Error in uploading file into repository

### DISCUSSION THREAD

**Discussion by hudsonhock** [community/community#126357] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi, I am uploading file (.exe) to my repository via "Attach binaries by dropping them here or selecting them", and getting error
![chrome_mWJuiNeJjr](https://github.com/community/community/assets/48878973/d8e20658-b262-456b-ac21-cde518ca88ae)
I compiled and uploaded these exe into Github so I can download later. These exe can be compile into x86 or x64, the problem is Github only accept x64 but flag error for x86 exe? I used to be able to upload these compiled exe into my Github repo without problem.

**Comment by judahpaul16** [comment]
I don't believe there any restrictions on architecture. However, if your file is larger than 100MB, you'll need to use [Git Large File Storage (LFS)](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage).

**Comment by math-silva** [comment]
Hey there,

Have you tried zipping the .exe file before uploading it? GitHub generally handles zipped executables without issue.

Let me know if that helps, and if it does, please mark this answer as accepted.

Cheers!

**Comment by hudsonhock** [comment]
@judahpaul16 I tried with Edge browser and upload both x86 and x64 compiled exe and the result remain the same, only x64 being accepted but x86 having error. The file is only 2MB size.
@math-silva I tried your suggestion, but it also rejecting the x86 zip file.
However, I tried other x86 exe which is not my compilation, and it has no problem to upload the file. This is getting weird, though!
I have raise this concern in the forum of the language I use to compile the exe to see if any input from their end.

**Comment by hudsonhock** [comment, marked as answer]
I found out the error is due to Zscaler (It’s a bundle of security services. ZScaler Internet Access proxies your Internet traffic and runs it through URL content filter lists.). It detect my x86 exe having trojan thus it restricted the upload.

**Comment by Bangsano** [comment]
When I uploaded a file to the repository, GitHub said an error so I couldn't upload the file to my repository

**Comment by RECITERSUNIVERSAL** [comment]
THE PROSITORY IS SHOWING ERROW

**Comment by RECITERSUNIVERSAL** [comment]
NOT ALOWING FILE UPLOAD
